inst/templates/package/testing_plan_package.Rmd

```diff
@@ -0,0 +1,42 @@
+---
+title: "Testing Plan"
+output:
+  word_document:
+    reference_docx: ../my-styles.dotx
+  pdf_document:
+    latex_engine: xelatex
+number_sections: yes
+params:
+  product_name: "package"
+  version: "v1.0.0"
+  auto_tests: ""
+  man_tests: ""
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE, results = "asis")
+library(knitr)
+library(flextable)
+```
+
+# Testing Plan: `r params$product_name` `r params$version`
+
+## Product Release Name or Number: `r params$product_name` `r params$version`
+
+## Scope
+The purpose of this Testing document is to outline the plan to test the Requirements outlined in the `r params$product_name` `r params$version` Requirements Specification document. Both Automated and Manual Testing at the unit level and End to End will be conducted to ensure that all Requirements have been met. Test results will be documented in the Test Results document. All Test results will be time stamped and attested by the tester. As new information becomes available during development, requirements may be added, removed or modified. As a result, corresponding tests will be updated to reflect the appropriate requirements. However, at the time of Validation, all tests will reflect the functionality outlined in the final Requirements Specification document.
+
+## Automated Testing
+Testing in both development and validation phases proceeded using a standardized, automated unit testing framework via the testthat R package. Tests were written in testthat format and saved to R source files located within the package repository. Testing was executed using the `testthat::test_check` function which ran all tests in every test file in the test directory. Each test was run in a clean R environment.  The test_check function returns a matrix of result data, with one line for each test and the corresponding result. The test matrix was retained and saved as an artifact from the validation testing. Tests relevant to the user stories covered by this change request were extracted from the larger test matrix to create the traceability matrix connecting the user story with the test result. 
+
+### Automated Tests
+The table below outlines the automated tests that will be conducted:
+```{r}
+format_auto_test_plan(params$auto_tests)
+```
+
+<!-- Manual tests -->
```

Comment:
I think we should just remove this section from the package docs, per other comments.